use_cases_and_horizontal_approaches/data_annotator_app/data_annotator_app.config.yaml

```diff
@@ -7,8 +7,6 @@ languages:
   - python
 authors:
   - Mel Hanna
```

Comment:
is Mel still at this company? Could we switch this to someone else?

Reply:
not anymore, can we leave her name as author? I will be the maintainer

Reply:
hm, let me check to see if that'll fail our test.

Reply:
it looks like those fields are optional, so we should be good